doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -114,6 +114,16 @@ nRF9160: Asset Tracker v2
 * Updated:
 
   * The application now uses the new LwM2M location assistance objects through the :ref:`lib_lwm2m_location_assistance` library.
+  * The application now uses the :ref:`lib_location` library for retrieving location information.
+    This is a major change in the application code although much smaller change from user perspective.
+    Here are additional notes about the change:
+    * Added location module and removed GNSS module. GNSS used through Location library.
+    * Neighbor cell handling moved from modem module to location module to be used through Location library
+    * Location module triggers location_request() with GNSS being 1st priority method and cellular 2nd priority if they are enabled in application configuration
+    * NMEA support removed. This was there only because nRF Cloud originally didn't support PVT but that has changed.
+    * AGPS/PGPS are not requested based on triggers in the application but only based on Location library events LOCATION_EVT_GNSS_ASSISTANCE_REQUEST and LOCATION_EVT_GNSS_PREDICTION_REQUEST
+    * LTE LC neighbor cell search type cannot be configured or defined with Location library at the moment. Default search type is always.
```

Comment:
```suggestion
    * LTE LC neighbor cell search type cannot be configured or defined with Location library at the moment. Default search type is "always".
```

Reply:
This sentence seems unfinished. I think it should have been:
`Default search type is always used.`
I've now modified it.

Reply:
Ah, ok!